File: testresources/java/accuracy/src/domain/indirect/allowedfrom/AccessInstanceVariableIndirect_VarVar.java
package domain.indirect.allowedfrom;

import domain.indirect.intermediate.BackgroundService;

public class AccessInstanceVariableIndirect_VarVar {
	private BackgroundService bckg;
	private String test;

	public AccessInstanceVariableIndirect_VarVar(){
		bckg = new BackgroundService();
	}
		// Indirect access of a normal attribute 
		public String case1() {
			test = bckg.serviceOne.name;
			return test;
		}
}

File: src/test/resources/csharp/benchmark/Domain/Indirect/ViolatingFrom/AccessInstanceVariableIndirect_VarVar.cs
namespace Domain.Indirect.ViolatingFrom
{
	public class AccessInstanceVariableIndirect_VarVar
	{
        private Domain.Indirect.Intermediate.BackgroundService bckg;

		private string test;
 
		public void Case1()
		{
            bckg = new Domain.Indirect.Intermediate.BackgroundService();
			test = bckg.serviceOne.name;
		}
	}
}
